system-addon/content-src/components/ComponentPerfTimer/ComponentPerfTimer.jsx

```diff
@@ -144,7 +146,9 @@ export class ComponentPerfTimer extends React.Component {
 
       this.props.dispatch(ac.SendToMain({
         type: at.SAVE_SESSION_PERF_DATA,
-        data
+        data,
+        // we don't want to trigger a redux update since this is triggered by render
+        meta: {skipLocal: true}
```

Comment:
It seems like SendToMain sending local by default is footgun due to it's non-intuitiveness.  Would it be a better solution to rename the current action SendToLocalAndMain and make SendToMain never send to local?

Reply:
I was actually just thinking about that, it would require some broader changes though.  How would you feel if I file a follow-up bug to address that?

Reply:
Isn't it just "search and replace" and then "add an action"?  Or is there more to the broader changes you describe?

Reply:
There are some actions which do actually need to be copied, so I'll need to make sure that those that do are changed appropriately and tested

Reply:
For example, blocking

Reply:
I guess the reason I'm hesistant to spin it off is that it's not like this thing has been a huge push, as far as I'm away, and all the mental context for coding and reviewing is here now.  I'll let you make the call.  If it's truly more work and worth the context-switch to pick up again later, than go ahead and spin it off.

Reply:
If we go down this route, then maybe `ac.OnlyToMain` and `ac.AlsoToMain`?

And do we actually want to use "skipOrigin"? It was added when Actions.jsm was initially implemented, but never used, but seems like what we want to use via various `actionCreators`: https://github.com/mozilla/activity-stream/blob/master/system-addon/common/Actions.jsm#L104

There is a skipMain only used by TopStoriesFeed: https://github.com/mozilla/activity-stream/search?q=skipMain and that could probably be `ac.OnlyToContent` (this happens to be a single content target.. but for broadcast… `ac.BroadcastToContent` -> `ac.AlsoToAllContent` and `ac.OnlyToAllContent`? Unless we want to specially call out single targets, e.g., `ac.OnlyToOneContent`, where that seems to match our current usage of preferring broadcast in the common case.)

Reply:
I like Also and Only since they don't conflict with the old names and clearly indicate what's going on. For Broadcast, we haven't really run into a usecase where that has to skip the main process, so dmose and I were thinking we can just leave as is